vhdbuilder/packer/install-dependencies.sh

```diff
@@ -459,6 +461,52 @@ elif [ "$OS" = "$MARINER_OS_NAME" ] && [ "$OS_VERSION" = "3.0" ] && [ "$(isARM64
   installAndConfigureArtifactStreaming acr-mirror-azurelinux3 rpm
 fi
 
+should_install_device_plugin() {
+  # Require x86_64 (skip arm64)
+  if [ "$(isARM64)" -eq 1 ]; then return 1; fi
+
+  # Ubuntu 24.04 only
+  if [ "$OS" = "$UBUNTU_OS_NAME" ] && [ "${UBUNTU_RELEASE:-}" = "24.04" ]; then return 0; fi
```

Comment:
Both Ubuntu 22.04 and 24.04 are supported.

Reply:
I want to avoid adding all combinations in one release/PR in case there are issues that are not caught by tests.

Reply:
Particularly given the huge usage of Ubuntu 22.04